### ISSUE TITLE
Eagerly load the first N images

### ISSUE THREAD

**Issue by verlok** [jdmdigital/JDM-Native-Lazy-Loading#3]
Hey there,

I strongly suggest to add the `loading=lazy` attribute only after the Nth image. 

The reason of that is when browsers will parse  the page and find the `loading=lazy` images, they will delay images download to a second stage, making rendering performance worse for the first images in the page, which we want to be rendered ASAP. 

The first images in the page should have the `loading=eager` attribute or no `loading` attribute at all (which defaults to `loading=auto`). 

The N number could be specified in the settings, or it could initially be set to an arbitrary number, e.g. `10`. 

Thanks for mentioning me and my article  in the tweet.

**Comment by jdmdigital** [comment]
That's a great suggestion @verlok.  I can't help but wonder if it _does_, in fact, make sense to postpone the rendering of images, even if they are above the fold, until a second pass.  We see this done all the time on sites like Medium where images in the body of the article are lazy-loaded, even the very first image at the top of the page (well-above the fold).

I like the idea of making it a setting.  Might create a super-simple settings page for the plugin and allow the website admin to make this determination for themselves.

**Comment by verlok** [comment]
Good idea. Go ahead!

**Comment by jdmdigital** [comment]
Thanks for your input.  We've added a new settings page in our latest commit... 113bfc1d6807e2964b8da3beec0cc2bd8a48b678
